apache/Dockerfile-alpine

```
@@ -33,15 +33,15 @@ RUN set -eux; \
     make \
     openssl \
     openssl-dev \
-    pcre-dev \
+    pcre2-dev \
     zlib-dev
 
 RUN set -eux; \
     wget --quiet https://github.com/owasp-modsecurity/ModSecurity/archive/refs/tags/v${MODSEC2_VERSION}.tar.gz; \
     tar -zxvf v${MODSEC2_VERSION}.tar.gz; \
     cd ModSecurity-${MODSEC2_VERSION}; \
     ./autogen.sh; \
-    ./configure --with-yajl --with-ssdeep --with-lmdb; \
+    ./configure --with-yajl --with-ssdeep --with-lmdb --with-pcre2; \
```

```suggestion
    ./configure --with-yajl --with-ssdeep --with-pcre2; \
```
mod_security2 module does not have `--with-lmdb` option.

---

That flag is currently only used in the Alpine builds. Is that a mistake? Shouldn't both nginx builds have the flag?